python/modules/IcePy/Proxy.cpp

```diff
@@ -31,7 +31,7 @@ namespace IcePy
 struct ProxyObject
 {
     PyObject_HEAD
-    shared_ptr<Ice::ObjectPrx>* proxy;
+    Ice::ObjectPrx* proxy;
```

Comment:
The principle is the same in Python, Ruby, PHP etc. We create a wrapper object that holds a *non null* proxy (ObjectPrx).